ccdproc/tests/test_image_collection.py

```diff
@@ -571,10 +567,6 @@ def test_setting_write_location_to_bad_dest_raises_error(self, tmpdir,
             for hdr in ic.headers(save_location=bad_directory.strpath):
                 pass
 
-    def test_no_fits_files_in_collection(self):
-        with catch_warnings(AstropyUserWarning) as warning_lines:
-            assert "no FITS files in the collection."
```

Comment:
This test makes no sense, so I just deleted it.

Reply:
Wow, going to be careful not to look at the blame on that one...

Reply:
You got me curious... It was added in #246 .

Reply:
Not sure how this ever actually did something....I'll have a word with the reviewer 😉